src/main/java/com/employee/info/mgt/app/controllers/main/MainController.java

```diff
@@ -45,11 +45,22 @@ protected void logButtonOnAction(ActionEvent event) {
 
         try {
             User currentUser = userFacade.findUserByUsername(username);
-            if (currentUser != null && password.equals(currentUser.getPassword()) || password2.equals(currentUser.getPassword())) {
-                showAlert("Login Successful", "Welcome " + username + "!", Alert.AlertType.INFORMATION);
-                openDashboardWindow(event);
-            } else {
-                showAlert("Login Failed", "Please double-check your username and password.", Alert.AlertType.ERROR);
+            if (username.isEmpty()) {
+                showAlert("Login Failed", "Please enter your username and password.", Alert.AlertType.ERROR);
```

Comment:
Why does it only check the username field if empty and then alert "Please enter your username and password."?

Reply:
> Why does it only check the username field if empty and then alert "Please enter your username and password."?

updated code already, thanks!